intelmq/bots/BOTS

```diff
@@ -653,6 +103,45 @@
                 "xmpp_server": "xmpp server",
                 "xmpp_user": "xmpp username"
             }
+        },
+        "AlienVault OTX": {
+            "description": "AlienVault OTX Collector is the bot responsible to get the report through the API. Report could vary according to subscriptions.",
+            "module": "intelmq.bots.collectors.alienvault_otx.collector",
+            "parameters": {
+                "api_key": "<insert your api key>",
+                "feed": "AlienVault OTX",
+                "rate_limit": "3600"
+            }
+        },
+        "BitSight Ciberfeed Stream": {
+            "description": "Bitsight Collector is the bot reponsible to get Bitsight Ciberfeed Alert Stream",
+            "module": "intelmq.bots.collectors.bitsight.collector",
+            "parameters": {
+                "feed": "BitSight",
+                "http_url": "http://alerts.bitsighttech.com:8080/stream?key=<api>"
+            }
+        },
+        "Blueliv Crimeserver": {
+            "description": "Blueliv Crimeserver Collector is the bot responsible to get the report through the API.",
+            "module": "intelmq.bots.collectors.blueliv.collector_crimeserver",
+            "parameters": {
+                "api_key": "<insert your api key>",
+                "feed": "Blueliv Crimeserver",
+                "rate_limit": "3600"
```

Comment:
This was like this before already. I only moved it, but of course I will change it.